### DISCUSSION TITLE
cannot render katex in side div element #8838

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#8839] [General] [answered]
<img width="928" alt="image" src="https://user-images.githubusercontent.com/20069410/228123738-79b1f34f-88b3-424b-bdf3-0700f196e193.png">
<img width="574" alt="image" src="https://user-images.githubusercontent.com/20069410/228123776-f7366f2f-8597-44d3-9282-6dfe7cd3fbd9.png">

**Comment by ghost** [comment]
<img width="755" alt="image" src="https://user-images.githubusercontent.com/20069410/228124108-51c196e9-ba4f-41e9-9da4-127df6a7a39e.png">
<img width="527" alt="image" src="https://user-images.githubusercontent.com/20069410/228124128-2051bcd2-9178-4028-a2a9-c3ec6077f773.png">

**Comment by Josh-Cena** [comment, marked as answer]
Please see the doc link I shared in https://github.com/facebook/docusaurus/issues/8838#issuecomment-1486159611.

Your first code doesn't work because there's no empty line between `<div>` and the Markdown, so the Markdown is interpreted as JSX, and `{ik}` becomes JSX syntax for interpolation.

Your second code doesn't work because you indented the Markdown by four spaces, making it become a code block.

The doc also recommends one way to write this kind of code,

```mdx
<div>

Content, without indentation

</div>
```

**Reply by ghost** [reply]
Adding empty line between div automatically adds p with padding. Can the p not be added?

**Comment by ghost** [comment]
Thank you, you are the best!
